recipe/cc_wrap.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
 
-$NN_CC_ORIG "$@" -mmacosx-version-min=13.3
+$NN_CC_ORIG "$@" -mmacosx-version-min=11.0
```

Comment:
To reduce the complexity of this branch, you can delete `cc_wrap.sh`, `cxx_wrap.sh`, and the code in `build-r-tiledbsoma.sh` that uses them. Those files are only needed for osx-* builds, and thus they have no effect on this branch to produce a linux-64 conda binary for the cloud env.

Reply:
Do you mind if I leave them in or do you actively want me to remove them? My thought was to keep the branch as sync with `main` as possible, but I can remove if you have a strong opinion either way.

Reply:
Your call. I find these diff's distracting because they suggest the changes are relevant to the PR, but they have zero effect on the build. But I see your point about keeping the two branches similar to make it easier to apply changes in parallel to the two branches in the future.